### PR TITLE
Requesting to add spocc download script

### PR DESCRIPTION
**Pull request by RandomForestz**
spocc_download.R contains code for download and visualizing species occurrence data from both the nic dataset and citizen science pulled from various databases (i.e., iNaturalist, GBIF). Download data will be added on the OneDrive.

### CODE REVIEW

File: spocc_download.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
+################################################################################
+################################################################################
+##################### Downloading Occurrences via SPOCC ########################
+################################################################################
+################################################################################
+
+# libraries
+library(spocc)
+library(sf)
+library(tidyverse)
```
Comment (Migfjim1) [inline spocc_download.R:10]: This is more of a overall pipeline question - do we want to load all our libraries in setup.R and then just run that at the top of every script?

**Review by ccmothes — APPROVED** [review]
We can merge this file as is, but in addition to my in-line comments here are a few more general things to work on over the next week or so:
- Since this is a workflow and not a function (I think I will want a single `get_occ()` function later on that does all the pulling and cleaning but not an immediate task), change this in to an .Rmd document that documents each steps and adds exploratory figures etc.
- Add some plots after pulling the `spocc` data, I'd like to see something like a faceted chart for each species with barplots showing the number of occurrences pulled from each databse (do this before removing duplicates).
- At the end of this workflow I'd like a single file for all species occurrences, so join the cleaned result from `spocc` with the NIC data and keep a column for the source of each occurrence.

File: spocc_download.R [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+foco_GMA <- sf::st_read("data/gis/ft_collins_GMA_boundary.shp")
+foco_GMA <- sf::st_transform(foco, st_crs(nic))
+
+# Plot
+plot(foco$geometry, axes = T) # boundary
+plot(nic["species"], add = T, pch = 16, pal = sf.colors(length(unique(nic$species))))
+
+
+# occ() // This can take some time with multiple species
+spp.listA = c("Agelaius phoeniceus", "Tyrannus verticalis", "Sturnella neglecta",
```
Comment (Migfjim1) [inline spocc_download.R:30]: As we think about broadening beyond the Nature in the City data, is it worth thinking through make this list without manually inputting species names? For example, we could store a csv of species and index it. E.g. for birds: https://checklist.americanornithology.org/ - not as familiar with butterflies.

Might just be more flexible, not as prone to error, and easier to add a long list as we expand this out.

**Review by Migfjim1 — COMMENTED** [review]
Not too many specific comments beyond what Caitlin has already pointed out. Looks good, Josh!

My only real substantive comment is about indexing a species list as we broaden this workflow out - see details in comment.

**Comment by ccmothes** [comment]
Adding one more comment I forgot about, in reference to converting the workflow to a .Rmd you can actually just build off of the `01_process_occurrences.Rmd` I already have in the root directory under the header at the bottom "Read and Process Additional Occurrence Data". So basically copy over the code in this R script and add the suggested edits in there.

**Pull request merged**